src/cluster_legacy.c

```diff
@@ -5362,6 +5363,21 @@ void clusterCloseAllSlots(void) {
  * Cluster state evaluation function
  * -------------------------------------------------------------------------- */
 
+void clusterLogFailReason(int reason) {
+    if (reason == CLUSTER_FAIL_NONE) return;
+
+    char *msg;
+    switch (reason) {
+    case CLUSTER_FAIL_NOT_FULL_COVERAGE:
+        msg = "At least one hash slot is not served by any available node. "
+              "Please check the 'cluster-require-full-coverage' configuration.";
+        break;
+    case CLUSTER_FAIL_MINORITY_PARTITION: msg = "I am part of a minority partition."; break;
```

Comment:
Different case, different style?

We can use same style in both cases.

```suggestion
    case CLUSTER_FAIL_MINORITY_PARTITION:
        msg = "I am part of a minority partition.";
        break;
```